openshift/deploy-scheduler-template.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: scheduler-babylon 
+  annotations:
+    openshift.io/display-name: "Scheduler Babylon Template" 
+    description: >-
+      Scheduler Babylon Template to install depploymentconfig, service and route,see
+      about using this template, including OpenShift considerations, see
+      https://github.com/redhat-gpe/scheduler/blob/master/README.adoc." 
+    tags: "go,golang,api" 
+    iconClass: icon-go-gopher 
+    openshift.io/provider-display-name: "Red Hat, Inc." 
+    openshift.io/documentation-url: "https://github.com/redhat-gpe/scheduler" 
+    openshift.io/support-url: "https://access.redhat.com" 
+objects:
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: scheduler-gpe
+  spec:
+    selector:
+      matchLabels:
+        app: scheduler
+      annotations:
+        # wait-for-ready used on BuildConfig ensures that template instantiation
+        # will fail immediately if build fails
+        template.alpha.openshift.io/wait-for-ready: "true"
```

Comment:
I don't think we want this annotation here. The comment refers to BuildConfig, which we are not using.